1_2.js

```diff
@@ -0,0 +1,10 @@
+while (true) {
+  var city = prompt("日本の首都は？");
```

Comment:
`var` は昨今ではコーディング規約で非推奨としている現場が多いので、利用を避けた方がいいです。

再代入が発生しうる場合は `let`
一度代入したら再代入をしない場合は `const` を利用しましょう

```suggestion
  const city = prompt("日本の首都は？");
```